source/ScreenSaverHero.hx

```diff
@@ -0,0 +1,67 @@
+package;
+
+import flixel.util.FlxSpriteUtil;
+import flixel.FlxG;
+import flixel.FlxSprite;
+
+class ScreenSaverHero extends FlxSprite
+{
+	var speed = 50;
+
+	public function new(x:Float = 0, y:Float = 0)
+	{
+		super(x, y);
+
+		makeGraphic(20, 10);
+		controlVelocity();
+	}
+
+	override function update(elapsed:Float)
```

Comment:
Now that we've got several different distinct kinds of functionality in update, it becomes very valuable to start breaking the code apart into helper methods.

For example, inside `update` we do the following:
 * set our `color` based on position
 * wrap around the boundaries of the screen
 * check keyboard inputs to change the angle and speed

Each of these operations is separate enough that they could live in their own function, similar to the `controlVelocity` method you created for your constructor.

Not required for merging the PR, but I recommend moving these different sections of code into their own private methods, then our update could end up being as clean as 
```haxe
override function update(elapsed:Float) {
    pickColor();
    FlxSpriteUtil.screenWrap(this);
    changeVelocity();
    super.update(elapsed);
}
```

Reply:
I'm currently working on this one and will return when I get some extra time.